shared/android/app/src/main/java/io/keybase/ossifrage/KBReactPackage.java

```diff
@@ -11,7 +11,6 @@
 import java.util.List;
 
 import io.keybase.ossifrage.components.KBTextInputManager;
-import io.keybase.ossifrage.modules.IntentHandler;
```

Comment:
So this is now done by the react module?

Reply:
You mean the intent data passing? That happens in main activity. See this comment: https://github.com/keybase/client/pull/18852/files#diff-270c1a26c4443ac1cfc4c6a7b223b13fR37